src/api/api.ts

```diff
@@ -38,6 +38,15 @@ type UkjentFeil = {
 
 export type Respons = IkkeLastet | LasterInn | Suksess | Feil | UkjentFeil;
 
+const transformerTilStilling = (data: any): Stilling => ({
+    ...data,
+    properties: {
+        ...data.properties,
+        workday: JSON.parse(data.properties.workday),
+        workhours: JSON.parse(data.properties.workhours),
```

Comment:
Disse to kommer som strings, så må parse dem etter å ha parset overordnet stilling